### ISSUE TITLE
Expected behavior post-view-worked-solution?

### ISSUE THREAD

**Issue by cjshawMIT** [wombats-writing-code/fbw-student-app#9]
Which wrong-answer LO is picked when the student views the worked solution on a target question? Currently there is no "wrong answer" ... so the system doesn't pick a follow-up question. What is the expected behavior?

**Comment by luwen-huang** [comment]
That's right. If the student chooses Show Me Answer on a target question, the system shouldn't pick a follow up question. Once Show Me the Answer is confirmed, the question is marked as red, and no longer able to submit (basically, done). 

are you returning something from backstage that will tell me explicitly if an item has Show'n me the Answer, or should we derive it client-side?

**Comment by cjshawMIT** [comment]
It will basically have a null response (no choice picked). Currently, you don’t need to derive it client-side, because the server won’t generate a new question (since there is no wrong answer LO).

**Comment by luwen-huang** [comment]
another question: asides from this scenario, is there any scenario when the server won't generate a next question AND there's no wrong answer LO?

**Comment by cjshawMIT** [comment]
Ahhh, that's a good question. We might have an issue with waypoint questions and show-me-the-answer. So only two cases where there is no wrong answer LO -- 1) show-me-the-answer (because there is no real answer submitted), and 2) not defined / the question is a fundamental question at the bottom of the tree. In the case of 1), no next question generated for a target question, but one might be for a waypoint question with the current configuration. In the case of 2), the next question generated will be at the same LO as the just-answered question.
